test/prima_auth0_ex/token_cache/dynamodb_test.exs

```diff
@@ -0,0 +1,25 @@
+defmodule Integration.TokenCache.DynamoDBTest do
+  alias PrimaAuth0Ex.TokenCache.DynamoDB
+
+  use PrimaAuth0Ex.TestSupport.TokenCacheBehaviorCaseTemplate,
+    async: true,
+    cache_module: DynamoDB,
+    # Token expiration is managed by aws, and could take days for old tokens to be deleted,
+    # so we don't cover that in the tests here
+    test_token_expiration: false
+
+  setup do
+    cache_env = Application.get_env(:prima_auth0_ex, :dynamodb_cache)
+
+    on_exit(fn ->
+      if cache_env == nil do
+        Application.delete_env(:prima_auth0_ex, :dynamodb_cache)
```

Comment:
Why delete if nil...? I don't understand the logic

Reply:
if it was nil, before we saved it we just reset it